maintenance.py

```diff
@@ -73,7 +73,7 @@ def make_rotating_tablename(prefix, delta=0):
     """Creates a tablename for table rotation based on a prefix with a given
     month delta."""
     date = get_month(delta=delta)
-    return "{}_{}_{}".format(prefix, date.year, date.month)
+    return "{}_{:04d}_{:02d}".format(prefix, date.year, date.month)
```

Comment:
+:100: on padding the year too. Not like we're going to regress anytime soon, but we were all here for Y2K.